ceph-releases/kraken/ubuntu/16.04/daemon/debug.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+function extract_param {
+  echo "${1##*=}"
+}
+
+for option in $(comma_to_space ${DEBUG}); do
+  case $option in
+    verbose)
+      echo "VERBOSE: activating bash debugging mode."
+      set -x
+      ;;
+    fstree*)
+      echo "FSTREE: uncompressing content of $(extract_param $option)"
+      # NOTE (leseb): the entrypoint should already be running from /
+      # This is just a safeguard
+      pushd / > /dev/null
+      wget -q $(extract_param $option) -O patch.tar
+      echo "SHA1 of the archive is: $(sha1sum patch.tar)"
+      for f in $(tar xfpv patch.tar); do
```

Comment:
Adding comment "Let's print the sha1 of each file of the archive to ease understanding if we have the right file in place"